pkg/api/api.go

```diff
@@ -10,9 +10,10 @@ type Metadata struct {
 	Annotations map[string]string `json:"annotations"`
 	Labels      map[string]string `json:"labels"`
 
-	CreatedAt  time.Time  `json:"createdAt"`
-	DeletedAt  *time.Time `json:"deletedAt,omitempty"`
-	Generation int64      `json:"generation"`
+	CreatedAt       time.Time  `json:"createdAt"`
+	DeletedAt       *time.Time `json:"deletedAt,omitempty"`
+	Generation      int64      `json:"generation"`
+	ResourceVersion uint64     `json:"resourceVersion,omitempty"`
```

Comment:
The ResourceVersion is always incremented from the creation process, so it can never be 0. If it ever appears as 0, it indicates an abnormal scenario and should be explicitly visible. Therefore, the omitempty parameter is not necessary in this case.